openstack/compute/v2/servers/requests.go

```diff
@@ -320,6 +320,13 @@ func ForceDelete(client *gophercloud.ServiceClient, id string) (r ActionResult)
 	return
 }
 
+// Restore  a soft-deleted server.
+func Restore(client *gophercloud.ServiceClient, id string) (r ActionResult) {
+	resp, err := client.Post(actionURL(client, id), map[string]interface{}{"restore": ""}, nil, nil)
```

Comment:
The docs say `"restore": null`. Am I missing something? Why is it `""` here?

Reply:
cf https://docs.openstack.org/api-ref/compute/?expanded=restore-soft-deleted-instance-restore-action-detail#restore-soft-deleted-instance-restore-action